### ISSUE TITLE
Warlock pet interupt will not appear if pet is summoned

### ISSUE THREAD

**Issue by cml57** [jordonwow/omnibar#247]
If enemy warlock has a felhunter active the icon will not appear on the omnibar when spell is used
I have spell ID 212619, ID 19647, ID 119910. I added 19647 to try to fix myself, it did not work.
If the warlock sacrifices their pet I am able to see the interrupt when used.

**Comment by bsjwow** [comment]
i have the same issue, any update?

**Comment by cml57** [comment]
no, it is still an issue... but Qing games today vs the same warlock I was not able to see the interrupt 1st game and was able to see the interrupt 2nd game and he had his warlock pet out both times
not sure what the issue it...
